src/claim_signer/GetClaimSigner.test.ts

```diff
@@ -0,0 +1,280 @@
+/* tslint:disable:no-relative-imports */
```

Comment:
We never standardized on directory name casing and sadly we're already using 
- PascalCase in the node, 
- kebab-case and camelCase in Frost API, 
- PascalCase and kebab-case in Frost Web and 
- PascalCase and lowercase in Explorer Web... 

don't know which is best, but I'd rather not introduce a new one.

Reply:
I was going to bring this up, but had no idea which type to suggest lol :D

Reply:
Pick one and let me know :D. That's years of ruby programming above for the directory names. It was always lower_snake_case. :)

Reply:
Hmmm I think there's only 1 file in this directory and its unit tests, right? If that's so, maybe it doesn't even need to be in its own directory... and we get to avoid this decision at the moment 😋

Reply:
Otherwise let's use PascalCase

Reply:
ack. I'm going to move it out to the upper directory, but will use PascalCase from now on (except when muscle memory kicks in 😆 )

Reply:
PascalCase can cause issues for cross-platform development (in NTFS, names preserve case, but are not case sensitive). Dashes and underscores are generally safe. I prefer `all-lowercase-dashed-filenames`, but will defer to whatever the consensus is among developers.

Reply:
That is a very good point. I ran into all sorts of issues with MySql around that, moving between Mac and Linux.

Reply:
I like `all-lower-case-dashed-names` for two reasons:
1. No Filesystem issues
2. Easier typing: no shift key usage

Reply:
For filenames, I like following the AirBnB JS standard.